### PR TITLE
salt pr

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
jk
makes it so you can't convert people to full cwc as a neutered cultist by removing the mania motor

## Why It's Good For The Game
kevinz oversight moment

## Changelog
:cl:
fix: chaplain cultists being able to convert people to full clockwork cult status
/:cl:

### CODE REVIEW

**Comment by Ludox235** [comment]
Wait people actually managed to make a mania motor? I tought about it and its conversion capacities but I gave up when I remembered that you need 2 people to make one and that clockwork marauders don't work.

Anyways that's funny as fuck.

**Comment by Arturlang** [comment]
Imagine: Two traitor chaplains.

**Comment by Sneakyrat6** [comment]
The chaplain probably could have just bought a second clockie doodad to make the mania motor with a friend.

File: code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm [hunk 1]
```diff
@@ -44,6 +44,7 @@
 	sort_priority = 2
 	quickbind = TRUE
 	quickbind_desc = "Creates a Mania Motor, which causes minor damage and negative mental effects in non-Servants."
+	var/requires_full_power = TRUE
```
Comment (Linzolle) [inline code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: no var/

Comment (ghost) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: holy shit i'm retarded

**Comment by ghost** [comment]
> The chaplain probably could have just bought a second clockie doodad to make the mania motor with a friend.

requires full power means neutered cultists cannot make it

**Pull request merged**